### ISSUE TITLE
User warning when cx_Freezing an application

### ISSUE THREAD

**Issue by anthony-tuininga** [marcelotduarte/cx_Freeze#78]
Originally reported by: **Jurko Gospodnetić (Bitbucket: [jurko](https://bitbucket.org/jurko), GitHub: [jurko](https://github.com/jurko))**

----------------------------------------

When freezing a simple application with the following content:

File: ``aaa.py``

```
#!python
print("Hello world")
import asyncio
import sys
print(sys.version_info)
```

File: ``setup.py``

```
#!python
from cx_Freeze import setup, Executable

setup(name="hhh",
      version="1.0",
      description="Haleluja!",
      options=dict(build_exe=dict(packages=[], excludes=[])),
      executables=[Executable("aaa.py", base="Console")])
```

I get the following ```UserWarning```:

```
#!text
C:\Program Files\Python\Python340\lib\site-packages\cx_freeze-4.3.3-py3.4-win-amd64.egg\cx_Freeze\freezer.py:585: UserWarning: Duplicate name: 'importlib/__init__.pyc'
  outFile.writestr(zinfo, data)
```

Environment:

* Windows 7 SP1 x64
* Python 3.4
* Microsoft Visual Studio 2010
* cx_Freeze based on current tip commit 5b9ee5b8e7deff348c0b15ba1e907c7bbf660667

This issue has been extracted from [comment 1415039](https://bitbucket.org/anthony_tuininga/cx_freeze/pull-request/47/wip-dont-freeze-importlib-into-base/diff#comment-1415039) in pull request #47.

----------------------------------------
- Bitbucket: https://bitbucket.org/anthony_tuininga/cx_freeze/issue/78

**Comment by anthony-tuininga** [comment]
*Original comment by* **foren junkie (Bitbucket: [lovetox](https://bitbucket.org/lovetox), GitHub: [lovetox](https://github.com/lovetox))**:

----------------------------------------

So i have this warning on freezing my application
in the zip file is 2 times the same file __init__.pyc, dont know how this is possible but it is.

how can i fix this?

**Comment by anthony-tuininga** [comment]
*Original comment by* **Anthony Tuininga (Bitbucket: [anthony_tuininga](https://bitbucket.org/anthony_tuininga), GitHub: Unknown)**:

----------------------------------------

This appears to be resolved in cx_Freeze 5.0.
